src/batch-client/index.ts

```diff
@@ -588,7 +588,7 @@ export class ZimbraBatchClient {
 			},
 			accountName,
 			singleRequest: true
-		}).then(Boolean);
+		}).then(res => normalize(CalendarItemCreateModifyRequest)(res));
```

Comment:
might be a risky change, please make sure to test Modern Ui core functionality

Reply:
I have created and modified the event  from the mail and calendar vertical and it has been created successfully.

Reply:
I guess this should not be required now

Reply:
Done. Removed the code.